deploy.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+echo "generating .gitHash.json"
+bash get_head.sh
```

Comment:
```suggestion
bash get_head.sh

```
*legendary nitpick* :P